portfolio/src/main/webapp/script.js

```diff
@@ -77,17 +77,18 @@ function showCaption(element) {
  * Fetch comments data from the data servlet to display on main page.
  */
 function getComments() {
-  fetch('/data')
+  const maxCommentsSelect = document.getElementById('max-comments');
+  var maxComments = maxCommentsSelect.value;
```

Comment:
You should change this variable to const since you are not changing its value. And in the instances that you would change the value, you should use let instead of var.

This is from our typescript style guide but still applicable:
Use const by default, unless a variable needs to be reassigned then use let. Never use var. const and let are block scoped, like variables in most other languages. var in JavaScript is function scoped, which can cause difficult to understand bugs. Don't use it.